Sources/Wendy/Docker/DockerCLI.swift

```diff
@@ -29,7 +29,55 @@ public struct DockerCLI: Sendable {
             error: .fileDescriptor(.standardError, closeAfterSpawningProcess: false)
         )
 
-        if !result.terminationStatus.isSuccess {
+        guard result.terminationStatus.isSuccess else {
+            throw SubprocessError.nonZeroExit(
+                command: ([self.command] + arguments).joined(separator: " "),
+                exitCode: Int(result.terminationStatus.description) ?? -1,
```

Comment:
does this actually convert to int? because it's a `.description`. is there another field? `esult.terminationStatus.exitCode` or something ?